CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ add_subdirectory(C/services/filter-plugin-interfaces/python/filter_ingest_pymodu
 add_subdirectory(C/services/north-plugin-interfaces/python)
 add_subdirectory(C/tasks/north)
 add_subdirectory(C/tasks/purge_system)
+add_subdirectory(C/tasks/update_alerts)
 add_subdirectory(C/tasks/statistics_history)
 add_subdirectory(C/plugins/utils)
 add_subdirectory(C/plugins/north/OMF)
```

Comment:
There seem to be a number of things here you don't need. All the north stuff, OMF, purge_system and statistics_history should not be required.

Reply:
This is main CMakeLists file which builds all the subdirectory. I just added update_alerts as well into it so that build process can build update_alerts directory as well